File: Frontend/src/app/people/donor.ts
import { BaseEntity } from '../classes/base-entity';

export class Donor extends BaseEntity {
  public notes: string;
  public status = DonorStatus.Unknown;
  public isBigDonor = false;
}

export enum DonorStatus {
  Unknown = 'Unknown',
  Active = 'Active',
  Inactive = 'Inactive'
}

export class Donation extends BaseEntity {
  public date: Date;
  public money: number;
  public personId: string;
  public missionOrgId: string;
}

File: Backend/Entities/Donor.cs
using LinqToDB;
using LinqToDB.Mapping;
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace Backend.Entities
{
    public class Donor : BaseEntity
    {
        public string Notes { get; set; }

        [Column(DataType = DataType.VarChar)]
        public DonorStatus Status { get; set; }

        public bool IsBigDonor { get; set; }
    }

    [JsonConverter(typeof(StringEnumConverter))]
    public enum DonorStatus
    {
        [MapValue(nameof(Unknown))]
        Unknown,

        [MapValue(nameof(Active))]
        Active,

        [MapValue(nameof(Inactive))]
        Inactive
    }
}